wurst/objediting/ObjUtil.wurst

```diff
@@ -0,0 +1,40 @@
+package ObjUtil
+
+import ObjectIds
+import Assets
+import LinkedList
+
+function asList(LinkedList<int> idList) returns string
```

Comment:
I'd prefer a generic solution that could be repurposed like so https://bin.wurstlang.org/wunomidoxe.wurst